### DISCUSSION TITLE
Capacity fade in drive cycle experiments

### DISCUSSION THREAD

**Discussion by mlnsai** [pybamm-team/PyBaMM#2161] [Q&A]
Hi PyBaMM team.

I have a couple of queries regarding drive cycle experiments. I am running a typical drive cycle scenario of driving around 18 miles a day and charging (0.1C) overnight, and i am running this cycle for an year.

1. The capacity fade ( in terms of Loss of Li inventory (%)) is same whether i drive 18 miles a day or 36 miles a day or even 72 miles a day for an year. Why isn't the fade more for longer miles driving condition?

2. The capacity fade is same even if I change the 0.1C slow charge to 1C charge overnight for the entire year. I was expecting a bit higher fade for increased charge rates. 

Where am i going wrong?

However, I am experiencing more fade as  i run the same cycle for 1 year, 2 years and even 5 years. But within a year, increased miles driven and increased charging rate is not affecting the fade. 

I am using SPMe model and enabled the following options.

`model = pybamm.lithium_ion.SPMe({"thermal": "lumped","SEI": "solvent-diffusion limited","SEI film resistance": "distributed", "SEI porosity change" : "true", "lithium plating": "reversible"})`

Please have a look at it. It would really help me. 

Thanks team :)

**Comment by valentinsulzer** [comment]
Under some conditions, SEI growth is dominated by calendar aging, i.e. it just depends on calendar time and not the cycling conditions. It depends on the parameters.

Before you go straight into the model with all these degradation effects, you should do a sensitivity analysis to better understand the effect of each parameter on each degradation mechanism
